scripts/helpers.sh

```diff
@@ -86,25 +87,24 @@ function start {
 
     export BLOCKCHAIN_URL="http://${ip}:8888"
     export SSO_WEBSITE_ORIGIN="http://${ip}:3000"
-    export REACT_APP_COMMUNICATION_URL="ws://${ip}:5000"
+    export VITE_COMMUNICATION_URL="ws://${ip}:5000"
     pm2 start npm --name "id" -- run start
 
     if [ "${ARG1}" == "all" ]
     then
-        export REACT_APP_SSO_WEBSITE_ORIGIN="${SSO_WEBSITE_ORIGIN}"
-        export REACT_APP_BLOCKCHAIN_URL="${BLOCKCHAIN_URL}"
+        export VITE_SSO_WEBSITE_ORIGIN="${SSO_WEBSITE_ORIGIN}"
+        export VITE_BLOCKCHAIN_URL="${BLOCKCHAIN_URL}"
         
-        echo "Starting Tonomy-ID-SSO-Website"
-        cd "${PARENT_PATH}/Tonomy-ID-SSO-Website"
-        BROWSER=none pm2 start npm --name "sso" -- start
-
-        echo "Starting Tonomy-ID-Demo-market.com"
-        cd "${PARENT_PATH}/Tonomy-ID-Demo-market.com"
-        BROWSER=none pm2 start npm --name "market" -- start
+        echo "Starting Tonomy-App-Websites"
+        cd "${PARENT_PATH}/Tonomy-App-Websites"
+        BROWSER=none pm2 start yarn --name "apps" -- dev --host
```

Comment:
I though you were going to run this several times on different ports? how will we access the different apps?

Reply:
instead of running this several times and losing performance on development device i mapped the ports with nginx alpine image